packages/beacon-node/src/network/reqresp/types.ts

```diff
@@ -92,13 +92,13 @@ export const responseSszTypeByMethod: {[K in ReqRespMethod]: ResponseTypeGetter<
   [ReqRespMethod.BlobSidecarsByRange]: () => ssz.deneb.BlobSidecar,
   [ReqRespMethod.BlobSidecarsByRoot]: () => ssz.deneb.BlobSidecar,
   [ReqRespMethod.LightClientBootstrap]: (fork) =>
-    ssz.allForksLightClient[onlyLightclientFork(fork)].LightClientBootstrap,
+    sszTypesFor<ForkLightClient>(onlyLightclientFork(fork)).LightClientBootstrap,
   [ReqRespMethod.LightClientUpdatesByRange]: (fork) =>
-    ssz.allForksLightClient[onlyLightclientFork(fork)].LightClientUpdate,
+    sszTypesFor<ForkLightClient>(onlyLightclientFork(fork)).LightClientUpdate,
   [ReqRespMethod.LightClientFinalityUpdate]: (fork) =>
-    ssz.allForksLightClient[onlyLightclientFork(fork)].LightClientFinalityUpdate,
+    sszTypesFor<ForkLightClient>(onlyLightclientFork(fork)).LightClientFinalityUpdate,
   [ReqRespMethod.LightClientOptimisticUpdate]: (fork) =>
-    ssz.allForksLightClient[onlyLightclientFork(fork)].LightClientOptimisticUpdate,
+    sszTypesFor<ForkLightClient>(onlyLightclientFork(fork)).LightClientOptimisticUpdate,
```

Comment:
are these generic params needed explicitly? They should be able to be inferred.

Reply:
Not required, can be inferred from the param.